_source/logzio_collections/_shippers/fluent-bit.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Fluent Bit
+logo:
+  logofile: fluent-bit.svg
+  orientation: vertical
+data-source: Fluent Bit
+open-source:
+  - title: Logz.io-Out Plugin for Fluent Bit
+    github-repo: fluent-bit-logzio-output
+contributors:
+  - imnotashrimp
+  - idohalevi
+shipper-tags:
+  - log-shipper
+---
+
+## Setup
+
+###### Configuration
+
+**You'll need**:
+CMake
+
```

Comment:
Why do we need the CMake note? It's the first line in FluentD installation instructions

Reply:
> I want to have the development part in the readme too. Not sure if it belongs to the docs or not -- your choice

Doesn't belong in the docs. This will go at the bottom of the readme, along with release notes

Reply:
It's out.